persistence/src/commonMain/kotlin/com/wire/kalium/persistence/daoKaliumDB/ServerConfigurationDAO.kt

```diff
@@ -1,9 +1,10 @@
-package com.wire.kalium.persistence.dao_kalium_db
+package com.wire.kalium.persistence.daoKaliumDB
```

Comment:
I'm not a big fan of messing cases styles in packages naming, not a blocker just an opinion, since we also have  `commonMain` `androidMain` and so on. I would name it just `daokaliumdb`